src/docs/get-started/install/_get-sdk-win.md

```diff
@@ -12,25 +19,52 @@
     (for example, `C:\src\flutter`; do not
     install Flutter in a directory like `C:\Program Files\` that requires
     elevated privileges).
- 1. Locate the file `flutter_console.bat` inside the `flutter` directory.
-    Start it by double-clicking.
+{% endcomment %}
+
+<ol markdown="1">
+<li markdown="1">At the command line,
+    go to the desired installation location
+    for the Flutter SDK (for example, `C:\src\flutter`; 
+    do not install Flutter in a directory like `C:\Program Files\`
+    that requires elevated privileges).
+</li>
+<li markdown="1">Clone the repo using `git clone`.
+    The following example clones the stable channel:
+
+```sh
+git clone -b stable https://github.com/flutter/flutter.git
+```
+</li>
+<li markdown="1">In the Finder, navigate to the `flutter`
```

Comment:
Finder? What kind of bizarre machine is this? I searched all over my Windows machine, but couldn't find a "Finder" anywhere. I think you mean, "Windows Explorer"...